jekyll/_cci2/orb-concepts.md

```diff
@@ -7,203 +7,171 @@ categories: [getting-started]
 
-Organizations are, by default, limited to claiming only one namespace. This policy is designed to limit name-squatting and namespace noise. If you require more than one namespace, please contact your account team at CircleCI.
-
-### Semantic Versioning in Orbs
-
-Orbs are published with the standard 3-number semantic versioning system:
-
-- major
-- minor
-- patch
-
-Orb authors should adhere to semantic versioning. Within config.yml, you may specify wildcard version ranges to resolve orbs. You may also use the special string volatile to pull in whatever the highest version number is at time your build runs.
+orbs:
+  <orb>: <orb>/<namespace>@x.y #orb version
 
-For example, when mynamespace/some-orb@8.2.0 exists, and mynamespace/some-orb@8.1.24 and mynamespace/some-orb@8.0.56 are published after 8.2.0, volatile will still refer to mynamespace/some-orb@8.2.0 as the highest semantic version.
+workflows:
+  use-orb-job:
+    jobs:
+      - <orb>/<job-name>
+```
 
-**Note:** CircleCI does not currently support non-numeric semantic versioning elements. We suggest that you use either semver-style version strings in x.y.z format, or a development-style version string in dev:* format.
+See the [Authoring Reusable Jobs]({{site.baseurl}}/2.0/reusing-config/#authoring-parameterized-jobs) guide for more information, and the [Using Node Test Job](https://circleci.com/orbs/registry/orb/circleci/node#usage-run_matrix_testing) example in the orb registry.
 
-Examples of orb version declarations and their meaning:
+## Namespaces
 
-* circleci/python@volatile - use the highest version of the Python orb in the registry at the time a build is triggered. This is likely the most recently published and least stable Python orb.
-* circleci/python@2 - use the latest version of version 2.x.y of the Python orb.
-* circleci/python@2.4 - use the latest version of version 2.4.x of the Python orb.
-* circleci/python@3.1.4 - use exactly version 3.1.4 of the Python orb.
+A _namespace_ is a unique identifier claimed by a user or organization to group a set of orbs by author. Each user or organization can claim _one_ unique and immutable namespace. Each namespace can contain many uniquely named orbs.
 
-## Orb Versions (Development vs. Production)
+For example, the `circleci/rails` orb may coexist in the registry with an orb called `<other-namespace>/rails` because they are in separate namespaces.
 
-There are two main types of orbs that you can use in your workflows: Development & Production. Depending on your workflow needs, you may choose to use either of these orbs. The sections below describe the differences between these two types of orbs so you can make a more informed decision of how best to utilize these orb types in your workflows.
+Organizations are, by default, limited to claiming only one namespace. This policy is designed to limit name-squatting and namespace noise. If you need to change your namespace, please contact your account team at CircleCI.
```

Comment:
> please contact your account team at CircleCI.
What if an individual doesn't have an account team? Maybe we switch this wording to "support team".